libs/ui/src/hooks/smartcard_auth/use_dipped_smartcard_auth.ts

```diff
@@ -23,9 +32,15 @@ import {
   parseUserFromCardSummary,
 } from './auth_helpers';
 
+interface DippedSmartcardAuthScope {
+  allowAdminsToAccessUnconfiguredMachines?: boolean;
```

Comment:
I need this toggle because admins can access unconfigured VxCentralScans but not unconfigured VxAdmins. I decided to place this toggle within the scope object because it's so related to the provided `electionDefinition`